lib/device.ex

```diff
@@ -15,11 +15,11 @@ defmodule Onvif.Device do
                 :serial_number,
                 :hardware_id,
                 :ntp,
-                media_service_path: "/onvif/services",
+                :media_ver10_service_path,
+                :media_ver20_service_path,
                 auth_type: :xml_auth,
                 time_diff_from_system_secs: 0,
                 port: 80,
-                supports_media2?: false,
```

Comment:
This is not needed now since we are storing the path for all the service versions. We can infer directly from the path if the particular service version is supported or not.